chall05/vkuikka.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/ruby
+#ruby 2.5.5p157
+
+require 'open-uri'
+
+if (!article = ARGV[0])
+	return
+end
+url = "https://en.wikipedia.org/wiki/"
+begin
+file = open(url + article)
```

Comment:
indentation issue here, even if it's not important in Ruby, always good to keep doing it